### PR TITLE
Review and clean-up

### PR DESCRIPTION
**Pull request by misterjones**
- Apply typical clean-up (typos, appease the linter, etc.).
- Add landing pages for "CX Design System" and its "Components" child page.

### CODE REVIEW

File: packages/cx-layout/doc/Header.md [hunk 1]
```diff
@@ -52,35 +53,37 @@ const Layout: FC = ({ children }) => (
...
 
 File to shadow:
 [`cxHeader.ts`](https://github.com/johnsonandjohnson/Bodiless-JS/blob/main/packages/cx-layout/src/components/Header/tokens/cxHeader.ts)
 
-#### Customing Via Extending
+#### Customizing Via Extending
 
-The site builder can create a new token that and override's with asHeaderToken() and then extend/override the specific domains.
+The Site Builder can create a new token that and overrides with `asHeaderToken()`, and then
+extends/overrides the specific domains.
```
Comment (misterjones) [inline packages/cx-layout/doc/Header.md]: @hvanyo I wasn't sure what should go here:

> ...create a new token that _______ and overrides with `asHeaderToken()`, ...

I'm guessing "extends"?

Comment (hvanyo) [reply packages/cx-layout/doc/Header.md]: The Site Builder can create a new token that utilizes `asHeaderToken()`, and then can
extends/overrides the specific domains within that token.

File: packages/cx-elements/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# CanvasX Design System
+
+<!--TODO: Provide overview of CanvasX Design System. -->
```
Comment (hvanyo) [inline packages/cx-elements/README.md]:
```suggestion
The CanvasX Design System is opinionated design that can aggregates suggested best practices for UX and performance to create optimal branded site. 
```

File: packages/cx-elements/doc/Components.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# CanvasX Components
+
+<!--TODO: Provide overview of CanvasX Components. -->
```
Comment (hvanyo) [inline packages/cx-elements/doc/Components.md]:
```suggestion
A collection of components that implement the CanvasX Design System.
```

File: packages/cx-layout/doc/Helmet.md [hunk 4]
```diff
@@ -1,54 +1,63 @@
...
-  - BodyHelmet: ?
+- `HreflangHelmet`: [Meta Hreflang Tag](https://moz.com/learn/seo/hreflang-tag)
+- `GtmHelmet`: Analytics scripts for [Google Tag Manager](https://tagmanager.google.com/)
+- `SeoHelmet`: For standard metadata such as meta description, robots tag, etc.
+- `SocialShareHelmet`: Used for [Open Graph](https://ogp.me/) and other social media, such as
+  Twitter card.
+- `LanguageHelmet`: Custom
+- `HTMLHelmet`: Will apply attributes to the `<html>` element that wraps the page. Includes things
+  such as direction, language, classes, etc.
+  <!-- TODO: Add description -->
```
Comment (hvanyo) [inline packages/cx-layout/doc/Helmet.md:14]: yeah I wasn't sure what this is used for. :) we never used it

**Review by hvanyo — APPROVED** [review]

**Review by hvanyo — APPROVED** [review]

**Pull request merged**